packages/kexec-tools/kexec-tools.spec

```
@@ -0,0 +1,32 @@
+Name: %{_cross_os}kexec-tools
+Version: 2.0.21
+Release: 1%{?dist}
+Summary: Linux tool to load kernels from the running system
+License: GPL-2.0
```

This is a deprecated SPDX identifier, and there's a mix of both licenses.

```suggestion
License: GPL-2.0-or-later AND GPL-2.0-only
```